Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1138 @@
+
+        return self.http_request_wrapper(
+            method="DELETE",
+            url_suffix=f"users/by-username/{email}/alerts/{alert_type}/{alert_id}",
+            return_empty_response=True
+        )
+
+    def get_alerts_last_week(self, email: str, portfolio_id: Optional[str]) -> Dict[str, Any]:
+
+        query_params: Dict[str, Any] = assign_params(
+            portfolio=portfolio_id
+        )
+
+        return self.http_request_wrapper(
+            method="GET",
+            url_suffix=f"users/by-username/{email}/notifications/recent",
+            params=query_params
+        )
+
+    def get_domain_services(self, domain: str) -> Dict[str, Any]:
+
+        return self.http_request_wrapper(
+            method='GET',
+            url_suffix=f"companies/{domain}/services"
+        )
+
+    def fetch_alerts(self, page_size: int, page: int) -> Dict[str, Any]:
+
+        query_params: Dict[str, Any] = assign_params(
+            username=self.username,
+            page_size=page_size,
+            sort="date",
+            order="asc",
+            page=page
+        )
+
+        return self.http_request_wrapper(
+            method="GET",
+            url_suffix=f"users/by-username/{self.username}/notifications/recent",
+            params=query_params
+        )
+
+    def http_request_wrapper(
+        self,
+        method: str,
+        url_suffix: Optional[str] = None,
+        params: Optional[dict] = None,
+        json_data: Optional[dict] = None,
+        return_empty_response: Optional[bool] = False
+    ):
+        """Wrapper for the ``http_request`` function
+
+        Args:
+            ``self`` (``SecurityScorecardClient``).
+            ``method`` (``str``): The HTTP method.
+            ``url_suffix`` (``Optional[str]``): The URL suffix, appended to the base URL. Defaults to None.
+            ``params`` (``Optional[dict]``): The query parameters sent in the HTTP request. Defaults to None.
+            ``json_data`` (``Optional[dict]``): The payload to be sent in the HTTP request in JSON format. Defaults to None.
+
+        Return:
+            ``dict`` or ``str`` or ``requests.Response``
+        """
+
+        return super()._http_request(
+            method=method,
+            url_suffix=url_suffix,
+            params=params,
+            json_data=json_data,
+            error_handler=self.error_handler,
+            return_empty_response=return_empty_response
+        )
+
+    @staticmethod
+    def error_handler(response: requests.Response):
+        """
+        Error handler for the API requests
+
+        Args:
+            response (requests.Response): The server's response to the HTTP request.
+        """
+
+        try:
+            error_response_json = response.json().get("error")
+            error_message: str = f'{error_response_json.get("message")} ({error_response_json.get("statusCode")})'
+            demisto.error(error_message)
```

Comment:
Why do we need this? you raise the error message and do return_error in the except in main.

Reply:
I took an example from [NetscoutArborSightline](https://github.com/demisto/content/blob/605689b202a7fe08a08992ec87d43a90d9443c9e/Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.py#L109-L131).

Reply:
I think this is not needed (even in netscout).

Reply:
removed in commit [d0bc62](https://github.com/demisto/content/pull/13582/commits/d0bc629bfcaf949d79e5461e2da90baa9627609b).